drivers/rtt_rtc/Makefile.dep

```diff
@@ -4,3 +4,4 @@ ifeq (,$(UNIT_TESTS))
 endif
 
 USEMODULE += rtc_utils
+USEMODULE += ztimer_no_periph_rtt
```

Comment:
This pulls in `ztimer`. But it's not required for this driver.

https://github.com/benpicco/RIOT/blob/drivers/rtt_rtc-ztimer/sys/ztimer/Makefile.dep#L35-L38

Reply:
good catch